docker-compose.yml

```diff
@@ -271,7 +271,7 @@ services:
       - ./config:/etc/georchestra
 
   elasticsearch:
-    image: elasticsearch:7.9.0
+    image: elasticsearch:7.15.1
```

Comment:
Newer versions seems to only be available under the docker image: `docker.elastic.co/elasticsearch/elasticsearch`.

So: `docker.elastic.co/elasticsearch/elasticsearch:7.15.1`

Reply:
right, I'll adapt the registry to the official elastic one, thanks

Reply:
I'd prefer to harmonize versions between georchestra's ansible playbook, docker ... and having the same server version as the client being used in geonetwork4.